src/index.html

```diff
@@ -22,6 +22,366 @@
+    </section>
+
+    <section class="product">
+      <div class="card" data-qa="card">
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">
+          Product code: 195434
+        </div>
+      </div>
+      <div class="rate">
+        <div class="rate--stars rate--stars-4">
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+        </div>
+        <div class="rate--reviews">
+          Reviews: 5
+        </div>
+      </div>
+      <div class="price">
+        <div class="price--label">
+          Price:
+        </div>
+        <div class="price--value">
+          $2,199
+        </div>
+      </div>
+      <button class="to-buy" data-qa="card-hover"> BUY </button>
+    </section>
+
+
+    <section class="product">
+      <div class="card" data-qa="card">
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">
+          Product code: 195434
+        </div>
+      </div>
+      <div class="rate">
+        <div class="rate--stars rate--stars-4">
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+        </div>
+        <div class="rate--reviews">
+          Reviews: 5
+        </div>
+      </div>
+      <div class="price">
+        <div class="price--label">
+          Price:
+        </div>
+        <div class="price--value">
+          $2,199
+        </div>
+      </div>
+      <button class="to-buy" data-qa="card-hover"> BUY </button>
+    </section>
+
+    <section class="product">
+      <div class="card" data-qa="card">
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">
+          Product code: 195434
+        </div>
+      </div>
+      <div class="rate">
+        <div class="rate--stars rate--stars-4">
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+        </div>
+        <div class="rate--reviews">
+          Reviews: 5
+        </div>
+      </div>
+      <div class="price">
+        <div class="price--label">
+          Price:
+        </div>
+        <div class="price--value">
+          $2,199
+        </div>
+      </div>
+      <button class="to-buy" data-qa="card-hover"> BUY </button>
+    </section>
```

Comment:
The product sections are repeated with identical content. Consider using a template system or JavaScript to dynamically generate this content to avoid redundancy and make maintenance easier.